src/InvestmentManager.sol

```diff
@@ -343,12 +343,12 @@ contract InvestmentManager is Auth {
         lpValues.maxMint = lpValues.maxMint + trancheTokensPayout;
         lpValues.remainingInvestOrder = remainingInvestOrder;
 
+        LiquidityPoolLike(liquidityPool).updatePrice(_toUint128(lpValues.depositPrice));
```

Comment:
I figured we should update with the deposit price here. As it is averaged for the user & is influenced by prev epochs. For the update, we would need to calculate a price only based on values from the executed order